<launch>
	<!-- knowledge interfaces -->
	<node name="rosplan_roadmap_server" pkg="rosplan_knowledge_base" type="rproadmapServer" respawn="false" output="screen">
	    <param name="data_path" value="$(find rosplan_planning_system)/common/" />
	    <param name="cost_map_topic" value="/move_base/local_costmap/costmap" />
	    <param name="odom_topic" value="/odom" />
	    <param name="use_static_map" value="false" />
	    <param name="static_map_service" value="/static_map" />
	</node>
</launch>
